test_runner/regress/test_download_extensions.py

```diff
@@ -113,6 +113,7 @@ def test_remote_extensions(
     cleanup(cleanup_files, cleanup_folders)
 
 
+"""
 # Test downloading remote library.
```

Comment:
there appears to be some error in this test
I think it's because postgis has libraries with weird names

Reply:
figured it out